File: frontend/src/app/models/album.model.ts
export class Album{
    album_name: string;
    album_price: number;
    album_artist: string;
    album_release_date: string;
    album_genre: string;
    cover_art:string;
    constructor(){
        this.album_name = '';
        this.album_artist = '';
        this.album_price = 0;
        this.album_release_date = '';
        this.album_genre = '';
        this.cover_art = '';
    }
}

File: backend/APIs/Models/AlbumModel.cs
namespace APIs.Models {
    public class AlbumModel {
        public Int64 album_id { get; set; }
        public string album_name { get; set; }

        public Int64 album_price { get; set; }

        public string album_release_date { get; set; }

        public string album_artist { get; set; }

        public string album_genre { get; set; }

        public string cover_art { get; set; }

    }
}
